src/pages/Metrics/Metrics.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import { PageHeaderWrapper } from '@ant-design/pro-layout';
 import { Empty, Button, Card } from 'antd';
 import { history } from 'umi';
+import { formatMessage } from 'umi';
```

Comment:
Also, please import useIntl instead of formatMessage, 

https://umijs.org/plugins/plugin-locale#useintl